modules/ROOT/pages/java-support.adoc

```diff
@@ -33,6 +33,25 @@ The maximum supported version for running your applications is Java 17 for Mule
 
 For more information about Mule release cadences, see xref:release-notes::mule-runtime/lts-edge-release-cadence.adoc[].
 
+== Steps for Upgrade 
+
+Follow these steps to upgrade to Java 17:
```

Comment:
what about reviewing custom code in all its form?

Reply:
Each of the links has info regarding custom code (custom connectors, custom policies, etc)